release_bot/celery_task.py

```diff
@@ -30,16 +31,36 @@ def parse_web_hook_payload(webhook_payload):
     Parse json webhook payload callback
     :param webhook_payload: json from github webhook
     """
+    db = get_redis_instance()
     if 'issue' in webhook_payload.keys():
         if webhook_payload['action'] == 'opened':
-            handle_issue(webhook_payload)
+            handle_issue(webhook_payload, db)
     elif 'pull_request' in webhook_payload.keys():
         if webhook_payload['action'] == 'closed':
             if webhook_payload['pull_request']['merged'] is True:
-                handle_pr(webhook_payload)
+                handle_pr(webhook_payload, db)
+    elif 'installation' in webhook_payload.keys():
+        if webhook_payload['action'] == 'added':  # detect new repo installation
+            installation_id = webhook_payload['installation']['id']
+            repositories_added = webhook_payload['repositories_added']
```

Comment:
isn't installation ID unique for every org/user? Wouldn't be enough to save `{org/username : ID}`

Reply:
I can't find documentation so I am not sure that this will be enough. Therefore, I saved the installation id for every repo. But I can change it can you send me some link to confirm your assumption? @rpitonak